### PR TITLE
Add aniket-24 to the list

### LINKED ISSUE #723: Aniket Kumar
screenshot-github com-2021 09 26-13_57_24

### PR DESCRIPTION
**Pull request by aniket-24**
resolve #723

### CODE REVIEW

**Comment by aniket-24** [comment]
@Panquesito7 please review and merge this!

File: data.json [hunk 1]
```diff
@@ -1349,4 +1349,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png",
     "issueNumber": 726
   }
```
Comment (deephunt3r) [inline data.json:1351]:
```suggestion
  }, 
```

Comment (deephunt3r) [reply data.json:1351]: Could you please make the required changes

File: data.json [hunk 2]
```diff
@@ -1349,4 +1349,10 @@
...
+  {
+    "name": "Aniket Kumar",
+    "githubUsername": "aniket-24",
+    "imageUrl": "https://user-images.githubusercontent.com/76562459/133885424-2f4af13d-239f-4e59-a2e7-e6324e275e55.png",
```
Comment (Panquesito7) [inline data.json:1355]: I would recommend more of an image like this.
image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134790539-719a671c-65d8-42b4-8ea7-3c2cf6f760bf.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by aniket-24 — COMMENTED** [review]
Now is it okay ? @deephunt3r

**Review by schmelto — APPROVED** [review]

**Comment by aniket-24** [comment]
@schmelto @Panquesito7 what all changes are required to merge this PR

**Comment by aniket-24** [comment]
Now, is it okay? @Panquesito7

File: data.json [hunk 3]
```diff
@@ -1354,5 +1354,11 @@
     "githubUsername": "hariketsheth",
     "imageUrl": "https://user-images.githubusercontent.com/72455881/134767118-d853cd39-ddc1-4df6-bda1-1a4315f9c855.png",
     "issueNumber": 728
+  },
+  {
+    "name": "Aniket Kumar",
+    "githubUsername": "aniket-24",
+    "imageUrl": "https://user-images.githubusercontent.com/76562459/134800047-8d5db274-3916-486f-9652-ea1e867cbd9d.png",
```
Comment (Panquesito7) [inline data.json:1361]: I would recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134812399-6a005576-3b24-46a6-b9bc-7798f79e5bf1.png",
```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍
Thanks for reviewing this pull request, folks! 🙂

**Pull request merged**